Formula/yadm.rb

```diff
@@ -0,0 +1,56 @@
+class Yadm < Formula
+  desc "Yet Another Dotfiles Manager"
+  homepage "https://github.com/TheLocehiliosan/yadm"
+  url "https://github.com/TheLocehiliosan/yadm/archive/1.04.tar.gz"
+  sha256 "a73aa51245866ce67aeb4322a62995ebbb13f29dc35508f486819dceb534968a"
+
+  def install
+    bin.install "yadm"
+    man1.install "yadm.1"
+  end
+
+  test do
+    # init a new repo
+    system "yadm", "init"
+    unless File.exist?("#{testpath}/.yadm/repo.git/config")
+      onoe "Failed to init repository"
+      return false
+    end
+
+    # confirm worktree set to $HOME
+    ohai "Verify repo worktree"
```

Comment:
Don't need to `ohai`. 95% of `brew test` runs are likely executed by our CI machines, so you're not really letting actual humans know what's coming up next.